docs/intro.rst

```diff
@@ -63,28 +69,43 @@ The ``mbdump-derived.tar.bz2`` archive contains annotations, user tags and searc
 These archives include all the data required for setting up an instance of
 CritiqueBrainz.
 
-You can automatically download the archives (``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2``) and
-begin the import for the MusicBrainz database::
+There are two ways in which you can import the database dump:-
 
-   $ docker-compose -f docker/docker-compose.dev.yml run musicbrainz_db
+1. By downloading and importing the database dump the data in a single command::
 
-.. note::
+    $ docker-compose -f docker/docker-compose.dev.yml run musicbrainz_db
+
+2. Or by manually downloading the dumps and then import it:-
+
+i. For this, you have to download the dumps ``mbdump.tar.bz2`` and ``mbdump-derived.tar.bz2``
+   from http://ftp.musicbrainz.org/pub/musicbrainz/data/fullexport/.
+
+   .. note::
+
+    Make sure to get the latest dumps
+
+ii. Then the environment variable ``DUMPS_DIR`` must be set to the path of the
+    folders containing the dumps. This can be done by::
 
-   An alternative way for setting up the MusicBrainz database is to
-   download the archives manually and then do the import. Archives provided from
-   https://musicbrainz.org can be downloaded from
-   https://musicbrainz.org/doc/MusicBrainz_Database/Download. Note that the
-   environment variable ``DUMPS_DIR`` must be set to the path containing the
-   downloaded archives. Then setup the MusicBrainz database using::
-
-      $ docker-compose -f docker/docker-compose.dev.yml run -v $DUMPS_DIR:/home/musicbrainz/dumps \
-      -v $PWD/data/mbdata:/var/lib/postgresql/data/pgdata musicbrainz_db
-
-   You can also use the smaller sample dumps available at
-   http://ftp.musicbrainz.org/pub/musicbrainz/data/sample/ to set up the MusicBrainz database.
-   However, note that these dumps are .tar.xz dumps while CritiqueBrainz currently only
-   supports import of .tar.bz2 dumps. So, a decompression of the sample dumps and recompression
-   into .tar.bz2 dumps will be needed.
+      $ export DUMPS_DIR="Path of the folder containing the dumps"
+
+    You can check that the variable ``DUMPS_DIR`` has been succesfully assigned or not by::
+
+      $ echo $DUMPS_DIR
+
+    This must display the path of your folder containing database dumps.
+
+3. Then import the database dumps by this command::
```

Comment:
this should be a subpart of 2 (by manually downloading the dumps and then import it).

Reply:
@paramsingh So what should I do? Add the manual dump installation part under a note column or just fix this thing?